mantle/kola/tests/misc/zincati-deadend.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2020 Red Hat
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package misc
+
+import (
+	"fmt"
+
+	"github.com/coreos/mantle/kola/cluster"
+	"github.com/coreos/mantle/kola/register"
+	"github.com/coreos/mantle/platform/conf"
+	"github.com/vincent-petithory/dataurl"
+)
+
+var graph = []byte(`{
+	"nodes": [
+	  {
+		"version": "${VERSION}",
+		"metadata": {
+		  "org.fedoraproject.coreos.scheme": "checksum",
+		  "org.fedoraproject.coreos.releases.age_index": "0",
+		  "org.fedoraproject.coreos.updates.deadend_reason": "https://github.com/coreos/fedora-coreos-tracker/issues/215",
+		  "org.fedoraproject.coreos.updates.deadend": "true"
+		},
+		"payload": "${PAYLOAD}"
+	  }
+	],
+	"edges": []
+  }`)
+
+var script = []byte(`#!/bin/bash
+  VERSION="$(/usr/bin/rpm-ostree status --json | jq '.deployments[0].version' -r)"
+  PAYLOAD="$(/usr/bin/rpm-ostree status --json | jq '.deployments[0].checksum' -r)"
+  sed -i 's/\"version\":.*/\"version\": \"'$VERSION'\"/g' /var/www/html/v1/graph
+  sed -i 's/\"payload\":.*/\"payload\": \"'$PAYLOAD'\"/g' /var/www/html/v1/graph`)
+
+var baseurl = []byte(`[cincinnati]
+base_url = "http://127.0.0.1:9001"`)
+
+var updates = []byte(`updates.enabled = true`)
+
+func init() {
+	register.RegisterTest(&register.Test{
+		Name:        "fcos.zincati.deadend.motd-info",
+		Run:         checkDeadEndReleaseInformation,
+		ClusterSize: 1,
+		Tags:        []string{"fcos", "zincati"},
```

Comment:
This looks like it should be restricted to just `fcos` with the addition of:

`Distros:              []string{"fcos"},`

Reply:
Would this be helpful? https://github.com/coreos/coreos-assembler/pull/1788/files#diff-057e18a98d5ca7d860bff8d17ff52eb28323ebe33348877a4754d589e7753611R109